client/src/modules/account_statement/modals/comment.modal.js

```diff
@@ -0,0 +1,41 @@
+angular.module('bhima.controllers')
+  .controller('CommentAccountStatementController', CommentAccountStatementController);
+
+// DI
+CommentAccountStatementController.$inject = [
+  '$uibModalInstance', 'data', 'AccountStatementService', 'NotifyService',
```

Comment:
What does data mean here? 

I see from the context of the controller calling it that it contains the selected rows from the grid - I think the name could better reflect what is happening here! 

Something like `gridSelectionContext`? Whatever makes sense to you.